src/ProgressWindow.cpp

```diff
@@ -68,6 +69,20 @@ ProgressWindow::UpdateLayout(PixelRect rc) noexcept
 
   bottom_position = rc;
   bottom_position.top = bottom_position.bottom - progress_border_height;
+  // For some reason (inconsistency with points and native pixels?)
+  // progress bar would be drawn below the window on iOS without correcting here
+  // ToDo: find real issue and fix there
```

Comment:
can you add your github handle to the Todo?

Reply:
Sure.